js/client/modules/@arangodb/testutils/instance-manager.js

```diff
@@ -334,6 +334,15 @@ class instanceManager {
       this.launchFinalize(startTime);
       return true;
     } catch (e) {
+      // disable watchdog
+      internal.SetGlobalExecutionDeadlineTo(0.0);
```

Comment:
should we also set `arangod.serverCrashedLocal`  there? or is this instance only?